docs/source/kernel-local.md

```diff
@@ -0,0 +1,28 @@
+## Local Deployment
+
+The Local deployment is just for local development and is not meant to be run in real life operations.
+
+If you just want to try EG in a local setup, you can use the following kernelspec:
+
+```json
+{
+  "display_name": "Python 3 Local",
+  "language": "python", 
+  "metadata": {
+    "process_proxy": {
+      "class_name": "enterprise_gateway.services.processproxies.processproxy.LocalProcessProxy"
```

Comment:
Just FYI (but might be worth mentioning), if Enterprise Gateway encounters a kernelspec without the `process_proxy` stanza, it will treat that kernelspec as if it contained `LocalProcessProxy`.  There are some basic "features" that local kernels can leverage from EG, but, for the most part, this is purely for backwards compatibility.

I agree that it makes sense to discuss this nonetheless - thank you!